mailbox/store/src/test/java/org/apache/james/mailbox/store/event/MailboxAnnotationListenerTest.java

```diff
@@ -77,7 +81,10 @@ public void setUp() throws Exception {
         eventFactory = new EventFactory();
         mailbox = new SimpleMailbox(MailboxPath.forUser("user", "name"), UID_VALIDITY, mailboxId);
 
-        deleteEvent = eventFactory.mailboxDeleted(mailboxSession, mailbox);
+        QuotaRoot quotaRoot = QuotaRoot.quotaRoot("root", Optional.empty());
+        QuotaCount quotaCount = QuotaCount.count(123);
+        QuotaSize quotaSize = QuotaSize.size(456);
```

Comment:
There are some quota fixtures, no?

Reply:
Those basic objects are in `james-core` or `mailbox-api` so they are able to all implementations.

That's just that not all implementations knows what to do with them.